library/src/main/java/com/novoda/downloadmanager/ExternalFilePersistence.java

```diff
@@ -140,30 +140,10 @@ public void delete(FilePath absoluteFilePath) {
         Log.d(getClass().getSimpleName(), message);
     }
 
-    @Override
-    public long getCurrentSize() {
-        if (fileOutputStream == null) {
-            Log.e("Cannot get the current file size, you must create the file first");
-            return 0;
-        }
-
-        try {
-            return fileOutputStream.getChannel().size();
-        } catch (IOException e) {
-            Log.e(e, "Error requesting file size, make sure you create one first");
-            return 0;
-        }
-    }
-
     @Override
     public long getCurrentSize(FilePath filePath) {
-        try {
-            FileOutputStream file = new FileOutputStream(filePath.path(), APPEND);
-            return file.getChannel().size();
-        } catch (IOException e) {
-            Log.e(e, "Error requesting file size for " + filePath);
-            return 0;
-        }
+        File file = new File(filePath.path());
```

Comment:
Changed this to be consistent with the `InternalFilePersistence`, this works and seems to be the preferred method.